### PR TITLE
Make key paths work regardless of working directory

### PR DESCRIPTION
**Pull request by RJPercival**
`RelativeToPackage(path)` will convert `path` to an absolute path. This means the key files can be found regardless of the location of test.

### CODE REVIEW

File: storage/testonly/admin_storage_tester.go [hunk 1]
```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/google/trillian"
 	spb "github.com/google/trillian/crypto/sigpb"
 	"github.com/google/trillian/storage"
+	testonly2 "github.com/google/trillian/testonly"
```
Comment (codingllama) [inline storage/testonly/admin_storage_tester.go]: nit: Could we find a better alias? `trilliantestonly`, `trltestonly`, `ttestonly`, something else?

File: testonly/integration/clientserver.go [hunk 2]
```diff
@@ -30,7 +30,8 @@ import (
 	"github.com/google/trillian/extension"
 	"github.com/google/trillian/server"
 	"github.com/google/trillian/storage/mysql"
-	"github.com/google/trillian/storage/testonly"
+	strtestonly "github.com/google/trillian/storage/testonly"
```
Comment (codingllama) [inline testonly/integration/clientserver.go]: nit: `str` reminds me of `string`. Maybe `stestonly`?

File: storage/testonly/admin_storage_tester.go (hunk 1 above)
Comment (RJPercival) [inline storage/testonly/admin_storage_tester.go]: Done.

File: testonly/integration/clientserver.go (hunk 2 above)
Comment (RJPercival) [inline testonly/integration/clientserver.go]: Done.

**Pull request merged**